index.Rmd

```diff
@@ -1,20 +1,49 @@
 --- 
-title: "Technical Guidelines"
+title: "Technical Guidelines for R"
 author: "Mirai Solutions"
 date: "`r Sys.time()`"
+description: "Best practices with R around select topics."
 github-repo: "miraisolutions/techguides"
 site: bookdown::bookdown_site
 output:
   bookdown::gitbook:
-    split_by: section
+    css: styles.css
+    includes:
+      in_header: target-blank.html
+    pandoc_args: ["--number-offset=0"]
     # disable alt text rendered as figure caption
+    split_by: section
     fig_caption: false
     config:
       sharing:
         github: yes
+      toc:
+        before: |
+          <li class="toc-logo"><a href="https://mirai-solutions.ch" target="_blank"><img src="mirai.png" alt="Mirai"></a></li>
+        after: |
+          <li><a href="https://github.com/rstudio/bookdown" target="_blank">Published with bookdown</a></li>
 ---
 
 # Introduction {-}
 
-[Mirai Solutions](https://mirai-solutions.ch) is happy to share technical
-guidelines, how-tos and best practices in different areas.
+```{bash mod-date, engine.opts='-l', echo=FALSE, comment=""}
+printf "Last modified: %s" "$(git log -1 --format=%cd index.Rmd | sed 's/^[^ ]* //;s/ [^ ]*$//')"
+```
+
+[**Mirai Solutions**](https://mirai-solutions.ch/) is a Zurich-based software development and consultancy firm, delivering cutting-edge technology and best practices to the industry, helping companies elevate their data analytics and operations.
+
+We are an interdisciplinary team of data scientists, software engineers, business consultants and IT architects with specialist knowledge ranging from finance and risk management to math/stats techniques to software development and project management.
+
+Our areas of expertise include Enterprise IT Architecture, Software Engineering & DevOps, Large-Scale Data Analytics, Financial Modeling and Training & Coaching.
+
+---
+
+![R Logo](Rlogo.png)
+
+In this e-book, we are happy to share technical guidelines, how-tos and best practices around R programming and DevOps in specific areas of interest.
+
+## Python guides
```

Comment:
```suggestion
**Python technical guides**
```

Reply:
addressed in [e004320](https://github.com/miraisolutions/techguides/pull/50/commits/e0043203619ca1730aff83c69d0080f1fbfc724a)